src/main/resources/migration/sql/001_Basic_tables.sql

```diff
@@ -0,0 +1,68 @@
+CREATE SCHEMA messaging;
+CREATE TABLE IF NOT EXISTS messaging.person
+(
+    person_id uuid NOT NULL,
+    name character varying(50) NOT NULL,
+    date_of_birth timestamp with time zone,
+    person_info character varying(300),
+    CONSTRAINT person_table_pk PRIMARY KEY (person_id)
+);
+COMMENT ON TABLE messaging.person IS 'Данные пользователя';
+COMMENT ON COLUMN messaging.person.person_info IS 'Информация в вольном стиле от пользователя о себе';
+CREATE TABLE IF NOT EXISTS messaging.dialog
```

Comment:
Do we need a `dialog` table if we have a `chat_room_messaging`?
As I can see the `dialog` is a specific case of `chat_room_messaging` (when we have only two participants)

Reply:
discussed, deleted